xml/cap_depl_install_caasp.xml

```diff
@@ -76,21 +76,48 @@
     <term>Network Requirements</term>
     <listitem>
      <para>
-      Your &kube; cluster needs its own domain and network. Each node should
-      resolve to its hostname, and to its fully-qualified domain name.
+      For &cf;'s nodes to be able to interact with each other, make sure your
+      network setup fulfills the following requirements:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        The &kube; cluster has an dedicated domain and subdomain.
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        Each node is able resolve to its host name and fully-qualified domain
+        name (FQDN).
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        &productname; has a dedicated domain or subdomain separate from the
+        &kube; cluster.
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        Traffic inside of &productname; resolves to a &kube; worker node in
+        that dedicated domain.
+       </para>
+      </listitem>
+     </itemizedlist>
+     <para>
       Typically, a &kube; cluster sits behind a load balancer, which also
       provides external access to the cluster. Another option is to use DNS
       round-robin to the &kube; workers to provide external access. It is also
       a common practice to create a wildcard DNS entry pointing to the domain,
-      for example *.&exampledomain;, so that applications can be deployed
+      for example <literal>*.&exampledomain;</literal>, so that applications
+      can be deployed
       without creating DNS entries for each application. This guide does not
       describe how to set up a load balancer or name services, as these depend
-      on customer requirements and existing network architectures.
+      on your requirements and existing network architectures.
      </para>
      <para>
-      <link xlink:href="https://www.suse.com/documentation/suse-caasp-3/book_caasp_deployment/data/sec_deploy_requirements_system.html#sec_deploy_requirements_network">
-      &suse; &caasp; Deployment Guide: Network Requirements</link> provides
-      guidance on network and name services configurations.
+      For information about network and name services configurations, see
+      <link xlink:href="https://www.suse.com/documentation/suse-caasp-3/book_caasp_deployment/data/sec_deploy_requirements_system.html#sec_deploy_requirements_network"/>.
```

Comment:
URL label is missing

Reply:
As it should be: https://doc.opensuse.org/documentation/styleguide/#sec.link

Reply:
Not using URL labels is bad SEO, and also not reader-friendly. If the link breaks, then readers can do a web search on the label, which in our manuals is usually some kind of document title. Also it looks icky. What is the reason for not using them?

Reply:
It comes out the same way in all output formats. Otherwise screen HTML looks nice but PDF and print-view HTML have ugly, super-long link texts because for both, we generate the link format `Link text (https://url.url.url)` or, in absence of a link title, just the URL: `https://url.url.url`).
I do agree that I should probably have added the book/section reference before the link however.

Reply:
ok thanks for explaining. We'll change all of them in a separate commit.